packages/components/src/Chips/ChipsTypes.tsx

```diff
@@ -50,19 +50,24 @@ export interface ChipSingleSelectProps extends ChipFoundationProps {
 
 export interface ChipMultiSelectProps extends ChipFoundationProps {
   readonly type: "multiselect";
-  readonly selected: string[];
+  readonly selected?: string[];
   onChange(value: string[]): void;
 }
 
 export interface ChipDismissibleProps extends ChipFoundationProps {
   readonly type: "dismissible";
-  readonly selected: string[];
+  readonly selected?: string[];
   onChange(value: string[]): void;
 
   /**
-   * Use a custom activator to trigger the Chip option selector
+   * Element that triggers opening the ChipDismissible Menu
    */
-  readonly activator?: ReactElement;
+  activator?: ReactElement;
+
+  /**
+   * Enables pass through of a data-testid. This is legacy functionality to allow old tests to pass.
```

Comment:
Oh let's match this description with the one we just updated 😄

Reply:
Donezo.